src/controller/CHIPDeviceController.h

```diff
@@ -611,6 +611,12 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController,
     // Commissioner will establish new device connections after PASE.
     OperationalDeviceProxy * GetDeviceSession(const PeerId & peerId) override;
 
+    // Issue an NOC chain using the associated OperationalCredentialsDelegate.
```

Comment:
This needs to document what format the NOC chain will be in.

Reply:
Fixed in https://github.com/project-chip/connectedhomeip/pull/20013